src/app/design/styles/color/index.html

```diff
@@ -487,6 +489,156 @@
   </div>
 	
 	</div>
+    
+    <div>
+  <stache-page-anchor>
+    Categorization
+  </stache-page-anchor>
+    <p>
+    Use categorization colors to categorize blocks of content by significant type. For example, use categorization colors for different types of events in calendars.
```

Comment:
It might just be me (and it might be too nitpicky to worry about), but the wording of `by significant type` sounds off to me, and it sees a little vague. What is a significant type? What do you think about changing `by significant type` to `based on shared types`?

Reply:
I think "significant" is more informative here. It's true that there's no reusable metric for "significance", but using that word at least _aims_ for the point that the distinction between the categories should be an important one for the user.

Reply:
Although, "based on" may be the clearer preposition here.

Reply:
Works for me. Thanks!

Reply:
Took another look at this because the tab was still open and mulled this over again. No sweat if you want to leave this as is, but what do you think of changing `by significant type` to `when the distinction between types of content is significant to users`? I think that might make this a little clearer.

Reply:
Done.